docs/discussion/DeletingAssertedSubClasses.md

```diff
@@ -0,0 +1,19 @@
+**NOTE** This documentation is incomplete, for now you may be better consulting the [GO Editor Docs](http://wiki.geneontology.org/index.php/Ontology_Editing_Guide)
```

Comment:
@dosumis - do you have any materials on using reasoner instead of asserting stuff? I would like to change this doc to that

Reply:
i think this will do as guidance for now.

Reply:
Just delete the GO Editor guide links